### DISCUSSION TITLE
Get Buyer Order Info

### DISCUSSION THREAD

**Discussion by alvarosalinero** [saleweaver/python-amazon-sp-api#545] [General]
Hi,

I am trying to get the information of the buyer of an order but I am not able to get anything.

Does anyone know what function returns this information?

Thank you

**Comment by saleweaver** [comment]
Do you have PII access?

**Reply by alvarosalinero** [reply]
I'm not sure.

Can you send me a screenshot of where it is made?

Thank you very much

**Reply by alvarosalinero** [reply]
Hi friend,

i can't find a way to grant that permission. Can you tell me how?

Thank you very much.

**Reply by sktong** [reply]
<img width="568" alt="Screen Shot 2022-05-13 at 18 19 56" src="https://user-images.githubusercontent.com/35784341/168264549-4c0bd87a-9f4b-4d30-a2c7-96313a913330.png">

Did you manage to check the restricted roles(see attached) when you registered your app?

**Reply by alvarosalinero** [reply]
Yes, I have it just like you
![amazon](https://user-images.githubusercontent.com/51154446/168265045-2eb56c38-d4df-45e7-b276-879a627b8a11.png)

**Reply by alvarosalinero** [reply]
Hello,

I have granted all the permissions but I still do not have access to all the information of the client.

Can you tell me what else I'm missing?

Thank you.

All the best,

Álvaro
